lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -172,8 +172,18 @@ bool CommandObjectDWIMPrint::DoExecute(StringRef command,
   {
     auto *exe_scope = m_exe_ctx.GetBestExecutionContextScope();
     ValueObjectSP valobj_sp;
-    ExpressionResults expr_result =
-        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options);
+    ExpressionResults expr_result = target.EvaluateExpression(
+        expr, exe_scope, valobj_sp, eval_options, &m_fixed_expression);
+
+    // Only mention Fix-Its if the command applies them.
+    // The compiler errors can address any parsing issues after applying
+    // Fix-It(s).
+    if (!m_fixed_expression.empty() && target.GetEnableNotifyAboutFixIts()) {
+      Stream &error_stream = result.GetErrorStream();
+      error_stream << "  Applying Fix-It to expression, changing it to:\n";
+      error_stream << "    " << m_fixed_expression << "\n";
```

Comment:
Doesn't LLDB have some abstractions to handle indentation? I thought I had seen something like this before, maybe someone else can chime in here

Reply:
Yeah, I wouldn't know about that yet, so someone else would need to chime in.

Reply:
> Oh I see other places in the old code do the same...

Yup!
So is this good to go as it is?

Reply:
I think this could be a useful thing to look at unifying in a follow-up patch.